seqr/views/utils/export_utils.py

```diff
@@ -77,10 +81,11 @@ def export_multiple_files(files, zip_filename, file_format='csv', add_header_pre
                 content += '\n'.join([
                     DELIMITERS[file_format].join([row.get(key) or blank_value for key in header]) for row in rows
                 ])
-                if not isinstance(content, unicode):
```

Comment:
If the content is a python 2 string, wouldn't this behavior change?

Reply:
if `content` is a python 2 `str` then `isinstance(content, str)` will be `False` because we import `str` from `future.builtins`. So, there will be no encoding but will be followed by a decoding (`str()`). This behavior is the same as that of old codes: `not isinstance(content, unicode)` is True, and then it will be decoded with `unicode()`.